components/event-feed-service/pkg/server/server.go

```diff
@@ -164,51 +173,41 @@ func FromInternalFormatToList(entries []*util.FeedEntry) ([]*event_feed.FeedEntr
 func validateFeedTimelineRequest(req *event_feed.FeedTimelineRequest) error {
 	// Validate Timezone
 	if req.Timezone == "" {
-		return errors.GrpcError(codes.InvalidArgument, "A timezone must be provided")
+		return e.GrpcError(codes.InvalidArgument, "A timezone must be provided")
 	}
 
 	_, err := time.LoadLocation(req.Timezone)
 	if err != nil {
-		return errors.GrpcErrorFromErr(codes.InvalidArgument, err)
+		return e.GrpcErrorFromErr(codes.InvalidArgument, err)
 	}
 
 	// Validate Interval
 	if req.Interval <= 0 || req.Interval > 24 {
-		return errors.GrpcError(codes.InvalidArgument,
+		return e.GrpcError(codes.InvalidArgument,
 			"Time interval must be greater than 0 and less than or equal to 24 hours")
 	}
 
 	if 24%req.Interval != 0 {
-		return errors.GrpcError(codes.InvalidArgument, "24 must be divisible by time interval")
+		return e.GrpcError(codes.InvalidArgument, "24 must be divisible by time interval")
```

Comment:
I think "time interval must divide 24" is a little easier to read, but anyone who gets this error is going to have to stare at it a bit anyway, so either is fine.

Reply:
:laughing: I didn't write :hocho:

Reply:
_carefully does not run `git blame`_